prisma/schema.prisma

```diff
@@ -71,6 +72,15 @@ model NotionMetadata {
   @@unique([contentId])
 }
 
+model YoutubeMetadata {
+  id         Int  @id @default(autoincrement())
```

Comment:
No need for db updates
if the url starts with `youtube.com`, then just use the youtube renderer